pkg/acsengine/const.go

```diff
@@ -15,12 +15,16 @@ const (
 	DefaultFirstConsecutiveKubernetesStaticIP = "10.240.255.5"
 	// DefaultAgentSubnetTemplate specifies a default agent subnet
 	DefaultAgentSubnetTemplate = "10.%d.0.0/16"
```

Comment:
Updated the default IP address count per network interface to 50, so that even the largest Kubernetes cluster with 12 pools * 100 nodes * 50 IP addresses = 60000 still fits a single /16 subnet. Added comments to the code to document how we decided on this default value.